### PR TITLE
Test: add review rating to CVPR test

### PR DESCRIPTION
**Pull request by celestemartinez**
This PR:
- adds secondary ACs support to custom stage
- adds review rating to CVPR test

### CODE REVIEW

File: openreview/stages/venue_stages.py [hunk 1]
```diff
@@ -1379,6 +1383,15 @@ def get_invitees(self, conference, number):
             invitees.append(conference.get_ethics_reviewers_id(number))
 
         return invitees
+    
+    def get_noninvitees(self, conference, number):
+        noninvitees = []
+
+        if conference.use_area_chairs and self.Participants.AREA_CHAIRS_ASSIGNED in self.invitees:
```
Comment (melisabok) [inline openreview/stages/venue_stages.py:1390]: this is can one single if, right?

File: openreview/stages/venue_stages.py [hunk 2]
```diff
@@ -1411,6 +1424,15 @@ def get_readers(self, conference, number):
             readers.append('${3/signatures}')
 
         return readers
+    
+    def get_nonreaders(self, conference, number):
+        nonreaders = []
+
+        if conference.use_area_chairs and self.Participants.AREA_CHAIRS_ASSIGNED in self.readers:
```
Comment (melisabok) [inline openreview/stages/venue_stages.py:1431]: so secondary area chairs won't see the ratings? are we sure about this?

Comment (celestemartinez) [reply openreview/stages/venue_stages.py:1431]: Well if we want Secondary ACs to be readers of ratings, we would only need to add them to readers in the CustomStage

**Comment by melisabok** [comment]
I think you need to fix the test and use a string for the rating instead of 0

**Comment by celestemartinez** [comment]
> I think you need to fix the test and use a string for the rating instead of 0

Yes! Sorry, I was using this to test Carlos' PR. This is fixed.

File: openreview/stages/venue_stages.py (hunk 1 above)
Comment (melisabok) [reply openreview/stages/venue_stages.py:1390]: do you want to unify this in a single if?

**Review by melisabok — APPROVED** [review]

**Pull request merged**